README.md

```diff
@@ -112,6 +114,8 @@ To install this library via [npm](https://www.npmjs.org) **version 3**, use the
 npm install @reactivex/rxjs
 ```
 
+This will include all builds for CJS/ES6/AMD/TypeScript.
```

Comment:
And clarify, 'including all builds for Typecript' is not correct term and I mentioned it in above comment that scoped package contains cjs/global only. Important thing for TS user is both package is being shipped with type definition can be used for typescript codebases.